src/components/RecipeModules.vue

```diff
@@ -143,8 +149,14 @@
 import atlasHelpers from "@/core/helpers";
 import type { Module } from "@/core/models";
 import * as yaml from "js-yaml";
+import { defineComponent } from "vue";
+import CopyBtn from "./CopyBtn.vue";
```

Comment:
it would be better to use a components loader instead of importing the component(s) every time is required somewhere

Reply:
I agree. I did it in the vos website, Atlas was not supposed to use components at all.